src/style.css

```diff
@@ -1 +1,50 @@
 /* add styles here */
+body {
+  display: flex;
+  margin: 0;
+  background-color: gray;
+}
+
+.display {
+  display: flex;
+  margin: 0 auto;
+  padding-bottom: 6px;
+  align-items: center;
+  background-color: rgba(255, 255, 255, 1);
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.stars__star {
+  width: 12px;
+  height: 12px;
+  background-image: url(images/star.svg);
+  background-size: cover;
+  margin: 2px;
+  margin-right: 6px;
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+  margin-bottom: 0;
```

Comment:
You have set 'margin-bottom: 0' on the last `.stars--5` child, which is good for overriding the default margin set on `.stars__star`. However, ensure that this is the only place where vertical margins are being manipulated to maintain consistency.

Reply:
Consider moving the margin bottom to the `:last-child` selector